src/cript/api/api.py

```diff
@@ -511,22 +510,42 @@ def save(self, project: Project) -> None:
 
         Returns
         -------
-        None
+        A set of extra saved node UUIDs.
             Just sends a `POST` or `Patch` request to the API
         """
+        if known_uuid is None:
+            known_uuid = set()
 
-        project.validate()
-
+        node.validate()
         # saves all the local files to cloud storage right before saving the Project node
         # Ensure that all file nodes have uploaded there payload before actual save.
-        for file_node in project.find_children({"node": ["File"]}):
+        for file_node in node.find_children({"node": ["File"]}):
```

Comment:
I think if we `use a send_post_request(node)` helper method then we can leave the current code as is and the user only thinks they are saving a project still

Reply:
see comment above.